OnlySwitch.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -86,17 +86,17 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/pointfreeco/swift-composable-architecture",
       "state" : {
-        "revision" : "a7c1f799b55ecb418f85094b142565834f7ee7c7",
-        "version" : "1.2.0"
+        "revision" : "3f80eb9ed07c16a9f42ac4f23d4dafaa0494568c",
+        "version" : "1.5.0"
```

Comment:
Is it more suitable to change it to version 1.5.6 to have minor bugs fixed? Or maybe we can change Dependency Rule from "exactVersion" to "upToNextMinorVersion"?

Reply:
1.5.6 has some breaking changes so I just updated it to 1.5.0 this time. 1.5.5 might be fine, but I didn't test it.